### PR TITLE
Small changes and updates to Getting Started docs

### PR DESCRIPTION
**Pull request by radinamatic**
## Summary

Adds a tip to address issues described in #1367, plus a couple of small changes.

### CODE REVIEW

**Comment by radinamatic** [comment]
@indirectlylit One thing I noticed recently is that Sphinx autobuild runs on the same local port as the server in development. I often run server in production when testing/documenting and haven't noticed it before, but happened to try to use the server in development this weekend and realized the conflict. Can we change something in the Sphinx build settings to make it run on a different port, or should I document this limitation?

**Comment by indirectlylit** [comment]
Auto-build takes a `--port` option: https://pypi.python.org/pypi/sphinx-autobuild

We can update our own documentations to recommend an alternate autobuild port, e.g.

```bash
sphinx-autobuild --port 8888 . _build
```

I think should work

**Comment by radinamatic** [comment]
Will test right away! :+1:

File: docs/dev/getting_started.rst [hunk 1]
```diff
@@ -125,6 +125,11 @@ To install Kolibri project-specific dependencies make sure you're in the ``kolib
     (kolibri)$ yarn install
 
 
+.. tip::
+
+  * We've adopted this concatenated version with added cleanup: ``make clean && pip install -r requirements.txt && pip install -e . && yarn install``.
```
Comment (indirectlylit) [inline docs/dev/getting_started.rst]: my newest version uses

```
pip install -r requirements.txt --upgrade
```

:)

Comment (radinamatic) [reply docs/dev/getting_started.rst]: Will be added! :wink:

**Comment by radinamatic** [comment]
Done & ready!

**Pull request merged**